src/result_processor.c

```diff
@@ -354,9 +354,6 @@ typedef struct {
     size_t nkeys;
     uint64_t ascendMap;
   } fieldcmp;
-
-  // return as soon as heap is full
-  bool quickExit;
```

Comment:
No longer needed since a pager is now being used instead of a sorter which can perform quick exit, right?

Reply:
Right. updated the PR description